src/components/footer/navigation-menu/navigation-menu.tsx

```diff
@@ -9,9 +12,25 @@ export const NavigationMenu = ({
   title,
   menuElements,
 }: NavigationMenuProps) => {
+  const [arrow, setArrow] = useState<string>("down");
```

Comment:
const [arrow, setArrow] = useState<string>("down");

Typescript can infer a type, it will know `arrow` is of type string because initial value is string "", so no need to explicitly type it as <string>. It is already string

Reply:
done